### ISSUE TITLE
Any plan for docs?

### ISSUE THREAD

**Issue by jiyinyiyong** [danielsz/meyvn#2]
Trying Meyvn since was mentioned in https://clojureverse.org/t/feature-request-release-to-clojars-with-clojurescript/2753/7 . And it's a litter better than Boot. I like it.

Got some questions.

* What I have folders other than `src/` containing my code? How do I tell Meyvn to bundle it?
* It appears deploying slow on my machine. It took 14s to finish deploying. Can I make it faster?

This is my current config https://github.com/Respo/inflow-popup/blob/dev/meyvn.edn

Do you have any plan for documentations?

**Comment by danielsz** [comment]
Your config looks good. I don't know if you can make this faster, or what that would entail. Source directories are derived from `deps.edn`, you can put there anything you want.

I am entertaining the idea to make this a commercial endeavor, which would mean that I would take documentation and optimization seriously, but for now you'll be better off taking it as it is, and submit PRs if you're so inclined.

**Comment by jiyinyiyong** [comment]
> Source directories are derived from deps.edn, you can put there anything you want.

I don't use `deps.edn` since my code is in ClojureScript. Can I write in `meyvn.edn`, which is the configuration file for Meyvn?

**Comment by danielsz** [comment]
No, use `deps.edn` for source and dependency directives. This will work with Clojurescript-only projects too. If you don't have one yet, go to your project directory and type at the command line:
```sh
$  echo "{:paths [\"mysourcedirectory\"]}" > deps.edn
```

**Comment by jiyinyiyong** [comment]
I'm giving up. Looked into code and it looks like it still a better option to have Meyvn reading a `:path` in `meyvn.edn` rather than in shadow-cljs. `:source-paths` in `shadow-cljs.edn` represents the code I want to run during development. Not necessarily the code I want to bundle into a jar.
